README.md

```diff
@@ -6,7 +6,7 @@
    [Download and install](https://nodejs.org/en/) if it necessary.
 2. Clone this repo.
 3. Install the basic project dependencies with the command `npm install`.
-4. Create .env.local file @ apps/freelance (near env.example file). Should include variables from example file with correct   values
+4. Create .env.local file @ apps/freelance (near env.example file). Should include variables from example file with correct values
```

Comment:
why you edit this file? why you push this file?

Reply:
fixed